doc/docs.md

```diff
@@ -2269,8 +2269,9 @@ $if debug {
 }
 ```
 
-If you want an `if` to be evaluated at compile time it must be prefixed with a `$` sign. Right now it can only be used to detect
-an OS or a `-debug` compilation option.
+If you want an `if` to be evaluated at compile time it must be prefixed with a `$` sign. Right now it can be used to detect
+an OS, compiler, platform or a `-debug` compilation option.
```

Comment:
or for [compile-time reflection](https://github.com/vlang/v/blob/master/doc/docs.md#compile-time-reflection)